pkgs/tools/misc/hyperledger-fabric/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  name = "hyperledger-fabric-${version}";
+  version = "1.3.0";
+
+  goPackagePath = "github.com/hyperledger/fabric";
+
+  # taken from https://github.com/hyperledger/fabric/blob/v1.3.0/Makefile#L108
+  subPackages = [
+    "common/tools/configtxgen"
+    "common/tools/configtxlator"
+    "common/tools/cryptogen"
+    "common/tools/idemixgen"
+    "cmd/discover"
+    "peer"
+    "orderer"
+  ];
+
+  src = fetchFromGitHub {
+    owner = "hyperledger";
+    repo = "fabric";
+    rev = "v${version}";
+    sha256 = "08qrrxzgkqg9v7n3y8f2vggyqx9j65wisxi17hrabz5mzaq299xs";
+  };
+
+  meta = with stdenv.lib; {
+    description = "An implementation of blockchain technology, leveraging familiar and proven technologies";
+    homepage = https://wiki.hyperledger.org/projects/Fabric;
+    license = licenses.asl20;
+    maintainers = [ maintainers.marsam ];
+    platforms = platforms.all;
```

Comment:
```suggestion

```

This is set automatically by `buildGoPackage`
https://github.com/NixOS/nixpkgs/blob/ca0c253a8066d32518e015e1befa27879fffcde0/pkgs/development/go-modules/generic/default.nix#L234